packages/extension/src/config/supportsPermitSignature.js

```diff
@@ -1,3 +1,3 @@
 export default {
-    '0x6B175474E89094C44Da98b954EedeAC495271d0F': true,
+    // '0x6B175474E89094C44Da98b954EedeAC495271d0F': true,
```

Comment:
We should probably re-enable this I think it got disabled at the hackathon as we hadn't upgraded

Reply:
This is DAI